docs/03-extensions/magento2/reference/axios-instances.mdx

```diff
@@ -10,7 +10,66 @@ sidebar_position: 1
 
 :::info WIP
 
-- `makeUserClientFromRequest`
 - `makeAdminClientFromRequest`
 
 :::info
+
+## `makeUserClientFromRequest`
```

Comment:
**thought:** I'm not sure to understand what the "user client" would be. It this to be differentiated with an "admin client"?

Reply:
It's to create a request for customers, for example fetching the cart does not require admin access